docker/depends/pecan.depends.R

```diff
@@ -98,7 +98,6 @@ wanted <- c(
 'RCurl',
 'REddyProc',
 'redland',
-'reshape',
 'reshape2',
```

Comment:
I am a little worried about this.  I have updated some code to use reshape2 and then ran the depends script

however if I grep the use of just reshape I see some packages still use that?

e.g. 
```
./modules/assim.sequential/inst/metfiltering.exploration.R:pre.dfm <- reshape::melt(pre.df, id = c("year"))
./modules/assim.sequential/inst/metfiltering.exploration.R:# tmp.dfm <- reshape::melt(tmp.df, id = c("year"))
```

but these really only look to be examples. So maybe thats OK?